miniProxy.php

```diff
@@ -138,7 +138,12 @@ function makeRequest($url) {
   curl_setopt($ch, CURLOPT_HEADER, true);
   curl_setopt($ch, CURLOPT_FOLLOWLOCATION, true);
   curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
-
+  
+  //Load a config for a website.
+  $domain = str_ireplace('www.', '', parse_url($url, PHP_URL_HOST));
+  if (file_exists(realpath("config/$domain.php"))) {   
+        include realpath("config/$domain.php");             
+    }
   //Set the request URL.
```

Comment:
[06-Mar-2018 11:11:28 UTC] PHP Fatal error:  Cannot redeclare proxifySrcset() (previously declared in /home/cabalpat/public_html/resurgencepatches/miniProxy.php:236) in /home/cabalpat/public_html/resurgencepatches/config/facebook.com.php on line 225

Reply:
But I didn't re-declare it...